### ISSUE TITLE
run the classifier on tensorflow GPU

### ISSUE THREAD

**Issue by amapic** [ardamavi/Dog-Cat-Classifier#2]
Hi,
Do you know if I can easily change your code to make it work on GPU ?

**Comment by ardamavi** [comment]
Yes you can with no any change in code, If you using  NVIDIA GPU, CUDA and TensorFlow GPU version.
Verify your GPU with `K.tensorflow_backend._get_available_gpus()` code (But first, you must import `keras.backend`).

If you want to use with multi GPU, import `keras.utils.multi_gpu_model` module and add `model = multi_gpu_model(model)` line to code.

**Comment by amapic** [comment]
Thank you !
